mounts/org.osbuild.ext4

```diff
@@ -26,16 +26,27 @@ SCHEMA_2 = """
   },
   "options": {
     "type": "object",
-    "additionalProperties": true
+    "properties": {
+      "mountopts": {
+        "description": "mounting options for the source",
+        "type": "array",
+        "items": {
+          "enum": ["ro"]
+        }
```

Comment:
You likely want `"minItems": 1`, or adjust your code to only append `-o` if the options array is non-empty.

Reply:
Wooo, empty array thing. Makes sense. Thanks!